MericCelik/assignment3/MainTest.java

```diff
@@ -0,0 +1,69 @@
+import org.junit.Test;
+import org.junit.Assert;
+import java.util.Arrays;
+import java.util.HashSet;
+
+/**@author AycaMericCelik
+ * */
+public class MainTest {
+
+    @Test (expected = NullPointerException.class)
```

Comment:
NPE is usually an unexpected error. How about using IllegalArgumentException or similar?

Reply:
Corrected! :+1: